packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -0,0 +1,210 @@
+<script lang="ts">
+import { faCaretDown, faCheck } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+
+export let id: string | undefined = undefined;
+export let name: string | undefined = undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let value: any = '';
+export let disabled: boolean = false;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let onInput: (val: any) => void = () => {};
```

Comment:
it's not clear to me what the `onInput` will do, is it more accurate to use `onChange` ?

Reply:
I had used onInput since that's what select had, but switched to more consistent onChange instead.